client/src/components/Search.jsx

```diff
@@ -4,12 +4,31 @@ import StarsModel from './StarsModel.jsx';
 export default class Search extends React.Component {
   constructor(props) {
     super(props);
+
+    let averageRatings = 0;
+    let starsLoaded = false;
+    if (props.ratings.length > 0) {
+      let sum = 0;
+      let obj = props.ratings[0];
+      for (var key in obj) {
+        sum += (obj[key] * 1)
+      }
+      averageRatings = sum / 6;
+      starsLoaded = true;
+    }
+    let totalRatings = 0;
+    let ratingsLoaded = false;
+    if (props.reviews.length > 0) {
+      totalRatings = props.reviews.length;
+      ratingsLoaded = true;
+    }
+
     this.state = {
       query: '',
-      starsLoaded: false,
-      avgRating: 0,
-      ratingsLoaded: false,
-      totalRatings: 0
+      starsLoaded: starsLoaded || false,
```

Comment:
@dweinzimmer, you no longer need these || logical operators because you are setting the initial value above and only updating if props contain what you need

Reply:
Good point, I've updated this now to remove the unnecessary `||` operators and tested that the code still works. Thanks!